gnosis/util/http.py

```diff
@@ -15,8 +15,7 @@ def prepare_http_session(
     session = requests.Session()
     retry_conf = (
         requests.adapters.Retry(
-            total=retry_count,
-            backoff_factor=0.3,
+            total=retry_count, backoff_factor=0.3, respect_retry_after_header=False
```

Comment:
Out of curiosity, wouldn't the server know better when we should attempt to retry the connection?

What are the use cases that you are thinking of where this decision should be on the client side?

Reply:
Maybe the service knows better in some scenarios. But for example some services tell you to retry in one hour, for example, blocking the request forever